example/integration_test/test_helpers.dart

```diff
@@ -0,0 +1,138 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:kiss_firebase_repository/kiss_firebase_repository.dart';
+import 'package:cloud_firestore/cloud_firestore.dart' as firestore;
+import 'package:firebase_core/firebase_core.dart';
+
+import '../../../kiss_repository/shared_test_logic/data/test_object.dart';
```

Comment:
if we can use package based imports

Reply:
yeah, it's like this now, cause it's a local import. 
That's a bit of a complication with the development.  I cannot reference remote kiss_repository as it's not published yet. Although I can reference github commit.. 

I'll do that